### PR TITLE
Taskbar customization 

### PR DESCRIPTION
**Pull request by j0rt3g4**
Solving #4076 
Added a sentence to refers also the taskbar customization
line 33

### CODE REVIEW

**Comment by illfated** [comment]
Also: replaces PR #4240

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/configuration/customize-windows-10-start-screens-by-using-mobile-device-management.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ In Windows 10 Pro, Windows 10 Enterprise, and Windows 10 Education, you can us
 >[!NOTE]
 >Support for applying a customized taskbar using MDM is added in Windows 10, version 1703.
 
-**Before you begin**: [Customize and export Start layout](customize-and-export-start-layout.md) for desktop editions.
+**Before you begin**: [Customize and export Start layout](customize-and-export-start-layout.md) for desktop editions. This preparation also works for taskbar customization.
```
Comment (nenonix) [inline windows/configuration/customize-windows-10-start-screens-by-using-mobile-device-management.md]:
```suggestion
**Before you begin**: [Customize and export Start layout](customize-and-export-start-layout.md) for desktop editions (also works for taskbar customization).
```

**Comment by illfated** [comment]
Looks like this is ready for final review and merge.

**Comment by mypil** [comment]
@ManikaDhiman -  PR has been copy edited and is ready for final review. This is based on issue #4076.

Thank you.

**Pull request merged**